AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/AnnotationPart.swift

```diff
@@ -0,0 +1,3 @@
+public protocol AnnotationPart: Codable {
+    var order: Int { get }
```

Comment:
Should we also add the properties that every `AnnotationPart` must have? 

```suggestion
    var id: UUID { get }
    var type: AnnotationType { get }
    var height: Double { get }
    var order: Int { get }
    var annotationId: UUID { get }
```